### PR TITLE
ui: Update message error

### PR DESCRIPTION
**Pull request by leonardojoao**

### CODE REVIEW

**Comment by leonardojoao** [comment]
@noreng-jg I finished this PR. Can you do review?

File: ui/src/components/snackbar/Snackbar.vue [hunk 1]
```diff
@@ -22,14 +23,38 @@ export default {
...
-    typeMessage: {
-      type: String,
-      default: 'default',
-      validator: (value) => ['default'].includes(value),
-    },
+  data() {
+    return {
+      mainContent: {
+        dashboard: 'dashboard',
+        notificationList: 'list notificarion',
```
Comment (noreng-jg) [inline ui/src/components/snackbar/Snackbar.vue:30]: typo

**Comment by leonardojoao** [comment]
@otavio sure, it's very good. Now, you can look at this change in the code.

**Comment by gustavosbarreto** [comment]
@all-contributors please add @leonardojoao  for code

**Pull request merged**